src/Page/Shared/View.elm

```diff
@@ -155,7 +155,7 @@ viewGuideTeaser teaser =
                     defaultTeaserImg
     in
     li [ css [ teaserContainerStyle ] ]
-        [ img [ src image.src, alt image.alt, css [ roundedCornerStyle, teaserImageStyle ] ] []
+        [ div [ alt image.alt, css [ backgroundImage (url image.src), roundedCornerStyle, teaserImageStyle ] ] []
```

Comment:
how well do screen readers deal with images that aren't `<img>`s? do they pick up on `title`/`alt` descriptors?

Reply:
good q! I don't know. @aaaaargZombies do you have any idea about this?

Reply:
It would seem [here](https://stackoverflow.com/questions/41942992/how-to-make-background-image-accessible) that generally an `<img>` should be used but you can make background images on `div`s accessible.

Reply:
Screen readers won't know these exist. Using background image is good for things that are decorative but would just add tedious rubbish into a screen reader or text-to-speech. So like on the towers project someone probably doesn't want to hear every time a new dash on the timeline appears.

You probably want some combination of these CSS properties on the `img`

```css
aspect-ratio: 1/1;
object-fit: cover;
object-position:  ??? // leaving this out will center it along the overflowing axis
```

for filler images generated by our `maybe` we should be adding `aria-hidden="true"` because these are just decorative.

Reply:
Oh nice! That works a treat. Have made those changes.